diff --git a/main.go b/main.go
index 7938d7f..4e8e81e 100644
--- a/main.go
+++ b/main.go
@@ -90,7 +90,7 @@ func shouldUseDDEKWin() bool {
 
 const (
 	cmdKWin             = "/usr/bin/kwin_no_scale"
-	cmdDdeSessionDaemon = "/usr/lib/deepin-daemon/dde-session-daemon"
+	cmdDdeSessionDaemon = "/usr/libexec/deepin-daemon/dde-session-daemon"
 	cmdDdeDock          = "/usr/bin/dde-dock"
 	cmdDdeDesktop       = "/usr/bin/dde-desktop"
 )
